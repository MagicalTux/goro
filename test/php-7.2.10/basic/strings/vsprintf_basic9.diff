     *** Testing vsprintf() : basic functionality - using hexadecimal format ***
     string(1) "b"
003- string(1) "B"
003+ string(2) "%X"
     string(4) "b 84"
005- string(4) "B 84"
005+ string(1) " "
     string(7) "b 84 b1"
007- string(7) "B 84 B1"
007+ string(2) "  "
     Done
