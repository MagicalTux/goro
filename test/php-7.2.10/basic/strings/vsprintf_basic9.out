*** Testing vsprintf() : basic functionality - using hexadecimal format ***
string(1) "b"
string(2) "%X"
string(4) "b 84"
string(1) " "
string(7) "b 84 b1"
string(2) "  "
Done